homeassistant/components/home_connect/api.py

```diff
@@ -369,4 +399,7 @@ def get_entity_info(self):
         """Get a dictionary with infos about the associated entities."""
         program_sensors = self.get_program_sensors()
         program_switches = self.get_program_switches()
-        return {"switch": program_switches, "sensor": program_sensors}
+        return {
+            "switch": program_switches,
+            "sensor": program_sensors,
+        }
```

Comment:
Revert this change.

Reply:
Done.

Reply:
Black is complaining on:   return {"switch": program_switches,"sensor": program_sensors}